src/Valit/Strategies/CompleteValitStrategy.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Valit.Strategies
+{
+    public class CompleteValitStrategy : IValitStrategy
+    {
+        public void Fail<TObject>(IValitRule<TObject> rule, IValitResult result, ref bool cancel) 
+            where TObject : class
+        {
+        }
+
+        public void Done(IValitResult result)
```

Comment:
I guess this one is just to mark that user can create custom behavior here, right?

Reply:
yup... I didn't want to put too many extensiblity points right away. But I think these two together would be useful in a Strategy.